djtriggers/tests/test_models.py

```diff
@@ -0,0 +1,106 @@
+from datetime import timedelta
+from logging import WARNING
+
+import mock
+import pytest
+from django.test import override_settings
+from django.test.testcases import TestCase
+from django.utils import timezone
+from locking.models import NonBlockingLock
+
+from djtriggers.exceptions import ProcessLaterError, AlreadyProcessedError
+from djtriggers.loggers.base import TriggerLogger
+from djtriggers.models import Trigger
+from djtriggers.tests.factories.triggers import NormalDummyTriggerFactory, RaisesExceptionDummyTriggerFactory
+
+
+class TriggerTest(TestCase):
+    """
+    Due to some bug in celery.contrib.methods.task_method testing the task method needs be done in the following way:
+        Class.TaskMethod(class_instance)
+    """
+
+    def test_handle_execution_success(self):
+        trigger = NormalDummyTriggerFactory()
+        trigger._handle_execution_success()
+
+        assert trigger.date_processed
+        assert timezone.now() - trigger.date_processed < timedelta(minutes=1)
```

Comment:
I'd prefer calling the trigger inside a fake_time block so we can make an exact assertion of the processed date